index.html

```diff
@@ -22,14 +22,14 @@
      </ul>
  </nav></div>
  <div class="demo">
-     <button class="btn1">request demo <span><img class="right-arr" src="right-arrow.svg" alt=""></span></button></div>
-     <div class="svg"><span><img class="svv" src="list.svg" alt=""></span>
+     <button class="btn1">request demo <span><img class="right-arr" src="./images/right-arrow.svg" alt=""></span></button></div>
+     <!-- <div class="svg"><span><img class="svv" src="list.svg" alt=""></span>
```

Comment:
the stuff i commented on is not necessary it for the mobile dropdown not needed yet